rdrf/report/migrations/0004_alter_reportdesign_options.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.2.18 on 2023-03-22 10:20
+from django.contrib.auth.models import Group, Permission
+from django.contrib.contenttypes.models import ContentType
+from django.db import migrations
+from django.db.models.functions import Lower
+
+from registry.groups import GROUPS as RDRF_GROUPS
+from report.models import ReportDesign
```

Comment:
This and some other imports aren't used anymore

Reply:
removed.